### PR TITLE
Fixed CLI and node-sass methods

### PR DESCRIPTION
**Pull request by andreamalvetta**

### CODE REVIEW

File: src/cli.js [hunk 1]
```diff
@@ -1,3 +1,6 @@
+const _ = require("lodash");
 const tokenImporter = require("./index");
 
-module.exports = tokenImporter();
+const options = _.includes(process.argv, '--convertCase') ? { convertCase: true } : {};
```
Comment (andreamalvetta) [inline src/cli.js:4]: I have added the argument `convertCase` to the CLI

File: src/index.js [hunk 2]
```diff
@@ -7,7 +7,10 @@ const extname = require("path").extname;
 const dirname = require("path").dirname;
 
 require("json5/lib/register"); // Enable JSON5 support
-// require("../utils/babel-register"); // To fix for CLI and node-sass methods
+
+if (process.title === 'node' || process.title === 'node-sass') {
+  require("../utils/babel-register");
+}
```
Comment (andreamalvetta) [inline src/index.js:13]: I require `@babel/register` only if I use CLI or node-sass method. If I use webpack i don't need to import this library

**Review by DanMad — APPROVED** [review]
This looks good to me, man. Awesome work.

The only question I have is "Does @babel/register need to be  a dependency?"

**Comment by andreamalvetta** [comment]
> This looks good to me, man. Awesome work.
> 
> The only question I have is "Does @babel/register need to be a dependency?"

Yes it does

**Pull request merged**